src/middleware/node/get-payload.ts

```diff
@@ -11,20 +11,16 @@ import AggregateError from "aggregate-error";
 // }
 type IncomingMessage = any;
 
-export function getPayload(request: IncomingMessage): Promise<string> {
-  // If request.body already exists we can stop here
-  // See https://github.com/octokit/webhooks.js/pull/23
-
-  if (request.body) return Promise.resolve(request.body);
-
+export function getPayload(request: IncomingMessage): Promise<Buffer> {
   return new Promise((resolve, reject) => {
-    let data = "";
-
-    request.setEncoding("utf8");
+    let data: Buffer[] = [];
 
     // istanbul ignore next
     request.on("error", (error: Error) => reject(new AggregateError([error])));
-    request.on("data", (chunk: string) => (data += chunk));
-    request.on("end", () => resolve(data));
+    request.on("data", data.push.bind(data));
```

Comment:
It's shorter and it's unlikely to break in future, but I'd still prefer we make it more explicit. The usage of `.bind` disappears from code since the use of arrow functions, fewer folks know what it is, how it works, and its drawbacks

```suggestion
    request.on("data", (chunk: string) => data.push(chunk);
```